source/gui/startupDialogs.py

```diff
@@ -153,9 +158,24 @@ def __init__(self, parent):
 		groupLabel = _("License Agreement")
 		sizer = wx.StaticBoxSizer(wx.VERTICAL, self, label=groupLabel)
 		sHelper.addItem(sizer)
-		licenseTextCtrl = wx.TextCtrl(self, size=(500, 400), style=wx.TE_MULTILINE | wx.TE_READONLY | wx.TE_RICH)
+		# Create a fake text control to determine appropriate width of license text box
+		_fakeTextCtrl = wx.StaticText(
+			self,
+			label="a" * 80,  # The GPL2 text of copying.txt wraps sentences at 80 characters
+		)
+		widthOfLicenseText = _fakeTextCtrl.Size[0]
+		_fakeTextCtrl.Destroy()
+		licenseTextCtrl = wx.TextCtrl(
+			self,
+			size=(widthOfLicenseText, self.scaleSize(300)),
```

Comment:
I believe the `wx.TextCtrl` also has a margin around the text. But the width is the total space the control takes up (including borders, margins, etc). Making it the width of 80 chars, may not make it big enough to contain 80 chars.
There might be a way to set the "clientSize".

Reply:
I think visually this does not matter, as the spacing provided for the license text ends up being greater than required to hold 80 chars.

Reply:
Note that the dialog appears the same for you, Quentin and I with vastly different display settings.